### ISSUE TITLE
Accessibility - Submit Screen - Why are Three Fields Not Updateable

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#43]
Why for the Region, Safety and Health Manager, and the Building fields are they grayed out along with already filled with content? VoiceOver is reading the content along with the words “dimmed text field” for each one. I would suggest the user be told they are disabled or can't be edited. If the user can’t enter anything for them, then why have them on the screen?

**Comment by aanthonyDOL** [comment]
For the 1st MVP , we grayed out Region, Safety and Health Manager, and the Building fields, because the 1st release of the app only targets Users in DOL building. Further more VoiceOver Text ("Text Field Cannot be Edited") was added to each greyed out text field.

**Comment by jfc3-dol** [comment]
Thanks, for the update. 

Closing issue.
